adapters/rest-vertx/src/main/java/org/eclipse/hono/adapter/rest/Config.java

```diff
@@ -68,8 +67,8 @@ public ApplicationConfigProperties applicationConfigProperties(){
      */
     @Bean
     @ConfigurationProperties(prefix = "hono.http")
-    public ServiceConfigProperties honoServerProperties() {
-        return new ServiceConfigProperties();
+    public RestProtocolAdapterProperties honoServerProperties() {
```

Comment:
Since the property class is absolutely identical to the MQTT adapter property class, I just introduced a common property class for protocol adapters on the main branch.
Can you be so kind and rebase the PR and eliminate then this class and change it to `ProtocolAdapterProperties`?
That creates shorter code without duplications....

Thanks.